test/JDBC/upgrade/13_5/schedule

```diff
@@ -48,4 +48,5 @@ sys-original_login
 sys-schema-name
 sys-suser_sid
 sys-suser_sname
-sys-trigger_nestlevel
\ No newline at end of file
+sys-trigger_nestlevel
+column_domain_usage
```

Comment:
how will this work? is this view available in 13.5 onwards?

Reply:
This view will be available in the next release. This test "column_domain_usage" should be present only in 14_latest schedule file.

Reply:
Right! we need to only update in "14_latest" base version. Updated in the next revision

Reply:
how come nothing failed in github actions?

Reply:
This is actually expected. The base version in this case is 13_5, prepare script runs in the base version. The verify script runs in the latest version which is  14_latest. So the tests are supposed to pass.